### PR TITLE
Mark non-interface OCI8 driver methods internal

### PR DESCRIPTION
**Pull request by morozov**
|      Q       |   A
|------------- | -----------
| Type         | deprecation
| BC Break     | no

See #4109.

### CODE REVIEW

**Review by greg0ire — DISMISSED** [review]
Do we usually add an UPGRADE note for that kind of change? Approving in case we don't

**Comment by morozov** [comment]
Yeah, these changes need to be added. Otherwise, there will be no record of these methods being refactored by `3.0.0`.

**Review by greg0ire — DISMISSED** [review]

File: UPGRADE.md [hunk 1]
```diff
@@ -1,5 +1,12 @@
 # Upgrade to 2.11
 
+## Non-interface driver methods have been marked internal
+
+The non-interface method of driver-level classes have been marked internal:
```
Comment (greg0ire) [inline UPGRADE.md:5]:
```suggestion
The non-interface methods of driver-level classes have been marked internal:
```

**Review by greg0ire — APPROVED** [review]

**Comment by greg0ire** [comment]
Thanks @morozov !

**Pull request merged**